src/main/java/com/jcraft/jsch/jce/Util.java

```diff
@@ -26,8 +26,8 @@
 
 package com.jcraft.jsch.jce;
 
-class Util {
-  static void bzero(byte[] foo) {
+public class Util {
```

Comment:
There is no reason for this class to be made public.
Please revert this change.

Reply:
needed to clean the copy

Reply:
This change is unneeded though to accomplish what you have proposed.
`src/main/java/com/jcraft/jsch/KeyPairECDSA.java` will be utilizing the class `src/main/java/com/jcraft/jsch/Util.java`, not `src/main/java/com/jcraft/jsch/jce/Util.java`.
Therefore there is no need for the changes to make `src/main/java/com/jcraft/jsch/jce/Util.java` public.